readme.md

```diff
@@ -59,64 +59,26 @@ df.file(__dirname, function (err, data) {
 
 ### df(callback)
 
-Get a list of filesystems with space info.
+Returns a promise for an array of filesystems with space info.
 
-#### callback(error, list)
+### df.fs(filesystem)
 
-*Required*  
-Type: `function`
-
-##### list
-
-Type: `array`
-
-List of `data` objects.
-
-### df.fs(filesystem, callback)
-
-Get space info for a specific filesystem.
+Returns a promise for an object with the space info for the specified filesystem.
 
 #### filesystem
 
 *Required*  
 Type: `string`
 
-#### callback(error, data)
-
-*Required*  
-Type: `function`
-
-##### data
-
-Type: `object`
-
-Data for the specified filesystem:
-
-- `filesystem` - The name of the filesystem.
-- `size` - Total size in bytes.
-- `used` - Used size in bytes.
-- `available` - Available size in bytes.
-- `capacity` - Capacity as a float from `0` to `1`.
-- `mountpoint` - Disk mount location.
```

Comment:
Just put it exactly how it is below `Returns a promise for an object with the space info for the specified filesystem.`